### DISCUSSION TITLE
RT-thread example "DataLogger" can not run on DUE board

### DISCUSSION THREAD

**Discussion by danilliu-sudo** [onelife/Arduino_RT-Thread#11] [Q&A] [answered]
hi, onelife. I tried to run the RT-thread example "DataLogger" on my DUE board ,but many compilation errors occured, such as 'O_ACCMODE' was not declared in this scope, 'open' was not declared in this scope, etc. 

what can I do to fix these? 

thanks for help!

**Comment by onelife** [comment, marked as answer]
Hi @danilliu-sudo,

This example saves sensor data into SD card, and is developed/tested by using `Adafruit 2.8" TFT Touch Shield v2`.
If having this hardware with you, please enable flag `CONFIG_USING_ADAFRUIT_TFT_CAPACITIVE` in `src/rtconfig.h` to compile this example. (Please disable `CONFIG_USING_MODULE` flag in the same file, if encountering issues.)
